package/patterns-yast.changes

```diff
@@ -1,3 +1,8 @@
+-------------------------------------------------------------------
+Mon Aug  9 11:56:10 UTC 2021 - Ladislav Slezák <lslezak@suse.cz>
+
```

Comment:
Is it ok not having a bug reference here?

Reply:
it's already in the previous change